package.json

```diff
@@ -228,5 +226,8 @@
     "vsce": "^1.100.1",
     "webpack": "^5.19.0",
     "webpack-cli": "^4.4.0"
+  },
+  "dependencies": {
+    "shell-quote": "^1.7.3"
```

Comment:
The type annotations are for a different version of this library.

Reply:
As far as I can tell, the public API hasn't changed between the patch versions - .2 was a regression fix, and .3 was a CVE fix.
There's no newer version of the @types either, so it should be fine like this.